doc/releases/release-notes-2.1.rst

```diff
@@ -2,25 +2,48 @@
 
 .. _zephyr_2.1:
 
-Zephyr 2.1.0 (Working Draft)
-############################
+Zephyr 2.1.0
+############
 
 We are pleased to announce the release of Zephyr kernel version 2.1.0.
 
 Major enhancements with this release include:
 
-* TBD
+* Normalized APIs across all architectures.
+* Expanded support for ARMv6-M architecture.
+* Added support for numerous new boards and shields
+* Added numerous new drivers and sensors.
+* Added new TCP stack implementation (experimental).
+* Numerous bug fixes to Bluetooth split software controller.
```

Comment:
does this really belong here as a major enhancement?

Reply:
Don't know. Was trying to capture something for this area. I'm completely open to suggestions here.

Reply:
done